src/go/k8s/tests/e2e/additional-cmdline-arguments/01-assert.yaml

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: redpanda.vectorized.io/v1alpha1
+kind: Cluster
+metadata:
+  name: cluster
+status:
+  replicas: 1
+  restarting: false
+  conditions:
+    - type: ClusterConfigured
+      status: "True"
```

Comment:
This should be moved to `00-assert.yaml` as we need to wait for `cluster` to report configured condition.